test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -63,11 +67,16 @@ public void testMissingColumnNameForFilepathMode() {
   @Test
   public void testMissingColumnNameForBlobMode() {
     thrown.expect(NullPointerException.class);
-    ResponseGenerator.blobColumn(Collections.<String, String>emptyMap());
+    ResponseGenerator.contentColumn(Collections.<String, String>emptyMap());
   }
 
-  /* Proxy based mock of ResultSet, because it has lots of methods to mock. */
   private ResultSet makeMockBlobResultSet(final byte b[]) {
+    return makeMockBlobResultSet(b, new ArrayList<String>(), new ArrayList<Integer>());
+  }
+
+  /* Proxy based mock of ResultSet, because it has lots of methods to mock. */
+  private ResultSet makeMockBlobResultSet(final byte b[], final ArrayList<String> names,
```

Comment:
Scrap the added "final" when you don't need them. Don't delete any existing ones, but this practice is "not favored".